code/game/objects/items/defibrillator.dm

```diff
@@ -1,25 +1,29 @@
 /obj/item/defibrillator
 	name = "emergency defibrillator"
-	desc = "A handheld emergency defibrillator, used to restore fibrillating patients. Can optionally bring people back from the dead."
+	desc = "A device that delivers powerful shocks to resuscitate incapacitated patients."
 	icon = 'icons/obj/items/defibrillator.dmi'
-	icon_state = "defib_full"
+	icon_state = "defib"
 	worn_icon_state = "defib"
 	atom_flags = CONDUCT
 	item_flags = NOBLUDGEON
 	equip_slot_flags = ITEM_SLOT_BELT
 	force = 5
 	throwforce = 6
 	w_class = WEIGHT_CLASS_NORMAL
-
+	///If the defibrillator is ready to use (paddles out)
 	var/ready = FALSE
-	///wether readying is needed
+	///Whether this defibrillator has to be turned on to use
 	var/ready_needed = TRUE
 	///The base healing number. This will be multiplied using DEFIBRILLATOR_HEALING_TIMES_SKILL.
 	var/damage_threshold = DEFIBRILLATOR_BASE_HEALING_VALUE
```

Comment:
probably just change this var name while you're at this. The name is entirely misleading to what it actually does

Reply:
still needs changing, this is a stinky var name